shared/chat/manage-channels/edit-channel-container.js

```diff
@@ -10,26 +10,36 @@ import {anyWaiting} from '../../constants/waiting'
 
 const mapStateToProps = (state: TypedState, {navigateUp, routePath, routeProps}) => {
   const conversationIDKey = routeProps.get('conversationIDKey')
-  const teamname =
-    routeProps.get('teamname') || Constants.getTeamNameFromConvID(state, conversationIDKey) || ''
-  const waitingForSave = anyWaiting(
+  if (!conversationIDKey) {
+    throw new Error('conversationIDKey unexpectedly empty')
+  }
+
+  const teamname = routeProps.get('teamname')
+  if (!teamname) {
```

Comment:
@chrisnojima mentioned that we want to reduce the info passed through routeprops, but it seems that teamname is necessary here -- if we don't have it, and it's not already in the teams data, we don't know which team to refresh to get it!